src/components/ProteinViewer/index.tsx

```diff
@@ -122,10 +124,39 @@ export const ProteinViewer = ({
   children,
 }: LoadedProps) => {
   const [isPrinting, setPrinting] = useState(false);
+
+  // State variable to show/hide "secondary" tracks
+  const [showMore, setShowMore] = useState(false);
+
+  // List of "main" tracks to be displayed, the rest are hidden by default
+  const mainTracks = [
+    'AlphaFold confidence',
+    'representative domains',
+    'representative families',
+    'variants',
+    'disordered regions',
+    'residues',
+  ];
+
   const [hideCategory, setHideCategory] = useState<CategoryVisibility>({
+    'secondary structure': true,
+    family: true,
+    domain: true,
+    'homologous superfamily': true,
+    repeat: true,
+    'conserved site': true,
+    'active site': true,
+    'binding site': true,
+    ptm: true,
+    unintegrated: true,
+    'other features': true,
     'other residues': true,
-    'external sources': true,
+    features: true,
+    predictions: true,
+    'match conservation': true,
+    'Clinical significance: pathogenic and likely pathogenic variants': true,
```

Comment:
```suggestion
    'pathogenic and likely pathogenic variants': true,
```

I don't think we need the prefix the track with Clinical significance.